doc/administrator/web/index.md

```diff
@@ -45,11 +45,16 @@ Servlet Container Configuration
 -------------------------------
 Two environment variables needs to be set in your Servlet container for IRIDA to function correctly: `spring.profiles.active=prod` and `irida.db.profile=prod`.
 
-You can adjust these variables in Tomcat by editing (depending on your distribution) `/etc/tomcat/tomcat.conf` (CentOS) or `/etc/default/tomcat7` (Ubuntu), and finding the `JAVA_OPTS` variable and setting the variables as shown below:
+You can adjust these variables in Tomcat by editing (depending on your distribution) `/etc/tomcat/tomcat.conf` (CentOS) or creating a `setenv.sh` within `/opt/tomcat/bin` (Ubuntu). For CentOS you would find the `JAVA_OPTS` variable and for `Ubuntu` you would add the variables as shown below:
 
 ```
 JAVA_OPTS="-Dspring.profiles.active=prod -Dirida.db.profile=prod"
 ```
+If these variables were added to `/opt/tomcat/bin/setenv.sh`, then you will need to make `setenv.sh` executable. This can be done by running the command below
+
+```
+chmod +X /opt/tomcat/bin/setenv.sh
```

Comment:
**suggestion:** I think this needs to be run as `sudo`

Reply:
Updated in 2cf8d9e0e195293418ba8ec9ac106bc07ff3cb68

Reply:
Thanks